Automattic/vip-go-mu-plugins
# Add index content validation

## Description

Adds in-depth ES index content validation helpers

Requires: https://github.com/Automattic/ElasticPress/pull/12

## Checklist

Please make sure the items below have been covered before requesting a review:

- n/a This change has relevant documentation additions / updates (if applicable).

## Steps to Test

1. Pull down PR
1. Index all content with `wp elasticpress index`
1. Disable VIP Search in the config (remove the constant)
1. Edit a post title and close that page in WP Admin (to prevent auto saves from interfering)
1. Re-enable search
1. Run the validation command - `wp vip-search health validate-contents`
1. Inconsistencies should be found and reported (post title or whatever was changed)
1. Re-index with `wp elasticpress index`
1. Re-validate, no inconsistencies should be reported
